docs/contributing/coding-guidelines/.pages

```diff
@@ -2,3 +2,4 @@ nav:
   - index.md
   - Language-specific coding guidelines: languages
   - ROS nodes coding guidelines: ros-nodes
+  - Message guidelines: message-guidelines
```

Comment:
It seems missing `.md`.
```suggestion
  - Message guidelines: message-guidelines.md
```

Also, I think you can place this under `ROS nodes coding guidelines`. (You can change the section name.)
I'll leave it to you.

Reply:
Thanks, I've fixed it. Also moved it under ros-nodes folder, it makes sense.

Reply:
Thank you! (I have to write other guidelines ASAP... :pleading_face: )